doc/release-notes.rst

```diff
@@ -12,12 +12,12 @@ versions of the Zephyr project are released approximately every three-months.
 All Zephyr project source code is maintained in a `GitHub repository`_;
 you can either download source as a tar.gz file (see the bottom of the
 `GitHub tagged releases`_ page corresponding to each release), or use
-Git clone and checkout commands, such as:
+Git checkout commands, on an existing Zephyr installation obtained using the
```

Comment:
On the line above this, there's mention of using the tar.gz file to get the source code.  Will that still work?  @nashif

Reply:
This won't work directly, no. I've modified this now to reflect reality.